core/src/main/java/com/coderanch/blackjack/Cards.java

```diff
@@ -61,4 +64,22 @@ private Cards() {}
     static Set<Card> getStandardDeck() {
         return STANDARD_DECK;
     }
+
+    static Stack<Card> getShuffledStandardDeck(){
```

Comment:
Don't use the mostly obsolete `Stack` class from the standard API. If you _really_ want to return a collection that exposes stack operations, return the `Deque` interface instead.

I strongly recommend returning a `Set` instead though, and let the calling method convert it to an ArrayDeque if they need stack operations. The reason for this is to keep it consistent with the `getStandardDeck` method, and to ensure that the returned collection contains only distinct cards.